x/farming/keeper/keeper.go

```diff
@@ -75,6 +76,17 @@ func NewKeeper(cdc codec.BinaryCodec, key sdk.StoreKey, paramSpace paramtypes.Su
 	}
 }
 
+// SetHooks sets the farming hooks.
+func (k *Keeper) SetHooks(fk types.FarmingHooks) *Keeper {
```

Comment:
Following general convention,

```suggestion
func (k *Keeper) SetHooks(fh types.FarmingHooks) *Keeper {
```

Reply:
Thanks! I will apply the suggestion.